server/src/response.rs

```diff
@@ -1457,6 +1457,10 @@ pub fn make_physical_mode(
         .map(|p| navitia_proto::PhysicalMode {
             uri: Some(format!("{}{}", PREFIX_ID_PHYSICAL_MODE, p.id)),
             name: Some(p.name.clone()),
+            co2_emission_rate: Some(navitia_proto::Co2EmissionRate {
+                value: p.co2_emission.map(|v| v as f64),
```

Comment:
```suggestion
                value: p.co2_emission.map(|v| f64::from(v)),
```
Numerical conversion with `as` do [different things in different context](https://doc.rust-lang.org/reference/expressions/operator-expr.html#numeric-cast) so I try to avoid it when possible.
Here the conversion is lossless, so you might as well use `from` so the reader don't worry about possible suprising behavior (truncation/overflow/rounding)

Reply:
Done in [f0fe50e](https://github.com/hove-io/loki/pull/350/commits/f0fe50e29b19c0a80d848ee2e1e376cd01a50d79).